src/client/AppState.ts

```diff
@@ -38,6 +38,7 @@ export const PRISTINE_APPSTATE: AppState = {
     vergunningen: [],
     registraties: [],
     daysLeft: 0,
+    title: 'Vakantieverhuur',
```

Comment:
Ik ben na enige overweging toch geen voorstander van het voorzien van de Thema naam via de api. Het schept een precedent voor toekomstige api's. Op basis van ervaringen uit het verleden is echter gebleken dat de api content niet 1-op-1 te koppelen is aan de thema namen. Zie bv stadspas gegevens die uit de focus api komen. Welke titel zou je daar neer moeten zetten?

Reply:
Prima, laten we het er morgen even over hebben.

Reply:
Deze mag weg.